server.go

```diff
@@ -151,6 +151,12 @@ func main() {
 		log.WithField("statsdUrl", s.StatsdUrl).Fatal("Could not connect to statsd")
 	}
 
+	if v := os.Getenv("EXCLUDED_TAGS"); v != "" {
+		for _, t := range strings.Split(v, ",") {
```

Comment:
Nothing wrong with this but I'm wondering if this is a good enough interface (in the rare(?) case when the number of excluded tags is large) – It's just becoming a bit of a hassle in route-master for instance 😅